Feature #049 fix graph y axix emoji

WooVictory opened the pull request:
## 개요
LineChart 그래프 y축 중복 문제 해결
## 작업사항
- y축 데이터의 max와 min 값을 5단계로 설정하고, 증가 간격을 1로 설정해서 중복 문제 해결.
- Presenter 안에서 데이터를 세팅하는 부분을 함수로 분리
- Model로부터 데이터를 받아서 처리하는 로직 미구현.
- resolved #49

WooVictory committed:
Merge branch 'feature_#023_setting_graph' into develop

WooVictory committed:
#000_변경 파일 커밋

WooVictory committed:
Merge branch 'develop' of https://github.com/boostcampth/boostcamp3_H into develop

WooVictory committed:
#049_fix_graph_y_axis
- LineChart 그래프 y축 이모지 중복 문제 해결
- y축 max / min 설정과 증가 간격 설정

WooVictory committed:
#049_fix_graph_y_axis
- runConfigurations file

KaquiIOS commented on app/src/main/java/team_h/boostcamp/myapplication/view/graph/GraphPresenter.java:
```diff
@@ -21,7 +20,7 @@
     public static final ObservableField<String> observer = new ObservableField<>("Statics");
     private List<Entry> entries = new ArrayList<>();
```
ResourceUtil에 getStringArray 만들어서 쓰시면 예쁠것같아요

WooVictory replied:
해당 부분 수정해서 올렸습니다.
https://github.com/boostcampth/boostcamp3_H/pull/59/commits/3cf3455a703a019f8e5906d956ca96c7425c7ef5

WooVictory committed:
#049_fix_graph_modify
- strings 파일에 string-array로 요일 설정
- ResourceSendUtil에 getStringArray 함수 생성
- GraphPresenter에서 getColor 수정 및 생성자에서 Context 제거
